.gitignore

```diff
@@ -1,3 +1,3 @@
 __pycache__
 data/*
-!data/.placeholder
+!data/config.ini
```

Comment:
I'm not sure having config file in the `data` directory. What would you think of having a `config` folder?